File: Praktikum Pemrograman I Modul 4/Praktikum I Modul4  07-11-2023/C/PRAK402-2310817110015-AKHMAD_CHAIDAR_ANANDA.c
#include<stdio.h>

int main(){

    // deklarasi variabel bilangan
    int bil;

    // input
    scanf("%d", &bil);

    // deklarasi variabel perulangan
    int i, j;
    // perulangan bilangan ganjil
    for (i = 1; i <= bil; i++){
        if (i % 2 == 1){
            printf("%d ", i);
        }
    }

    printf("\n");

    // perulangan bilangan genap
    for (j = bil; j >= 1; j--){
        if (j % 2 == 0){
            printf("%d ", j);
        }
    }

    return 0;
}

File: Praktikum Pemrograman I Modul 4/Praktikum I Modul4  07-11-2023/Python/PRAK402-2310817110015-AKHMAD_CHAIDAR_ANANDA.py
# input
bil = int(input())

# perulangan untuk bilangan ganjil
for i in range(1, bil + 1):
    # kondisi, jika input bilangan ganjil
    if (i % 2 == 1):
        print(f"{i}", end=' ')

print("")

# perulangan untuk bilangan genap
for j in reversed(range(1, bil + 1)):
    # kondisi, jika input bilangan genap
    if (j % 2 == 0):
        print(f"{j}", end=' ')